components/device-mgt/org.wso2.carbon.device.mgt.mobile.impl/src/main/java/org/wso2/carbon/device/mgt/mobile/impl/windows/dao/WindowsDAOFactory.java

```
@@ -104,14 +116,13 @@ public static void commitTransaction() throws MobileDeviceManagementDAOException
                 }
             }
         } catch (SQLException e) {
-            throw new MobileDeviceManagementDAOException("Error occurred while committing the transaction", e);
+            log.error("Error occurred while committing the transaction", e);
```

Is it good to hide this exception of commit at this level by catching the exception and just logging without throwing?
With this, db commit failures will go hidden at the caller level and a proper rollback would not happen,